vortex-datafusion/src/persistent/execution.rs

```diff
@@ -28,19 +27,17 @@ impl VortexExec {
         projection: Option<&Vec<usize>>,
         predicate: Option<Arc<dyn PhysicalExpr>>,
     ) -> DFResult<Self> {
-        let partitioning = Partitioning::UnknownPartitioning(1);
+        let projected_schema = project_schema(&file_scan_config.file_schema, projection)?;
         let plan_properties = PlanProperties::new(
-            EquivalenceProperties::new(file_scan_config.file_schema.clone()),
-            partitioning,
+            EquivalenceProperties::new(projected_schema),
+            Partitioning::UnknownPartitioning(1),
```

Comment:
we should figure this one out. Requires some extra config in the reader

Reply:
Right now, we do only use a single partition in the file scan as all files are in the same group.